create_release_pull_request/action.yml

```diff
@@ -84,12 +84,19 @@ runs:
         git checkout -b $RELEASE_BRANCH
         git push origin $RELEASE_BRANCH -f
 
+        # Add milestone if available
+        if [ -n "${{ steps.find_milestone.outputs.milestone_url }}" ]; then
```

Comment:
FYI: if `milestone_url` is found, that means the milestone from the release version is available from the repo.